src/redux/reducers/examplesOptions.ts

```diff
@@ -62,37 +56,37 @@ export const examplesOptionsReducer = (
           ...state,
           exampleFirstOptions: {
             ...state.exampleFirstOptions,
-            data: exampleFirstInitialValueRu.data,
-            options: exampleFirstInitialValueRu.options,
+            data: getExampleFirstInitialValue(Language.Ru).data,
+            options: getExampleFirstInitialValue(Language.Ru).options,
```

Comment:
Can it be simplified smth like:
data: getExampleFirstInitialValue(action.payload).data,
options: getExampleFirstInitialValue(action.payload).options,

in this case we can reduce duplicated code

Reply:
I could have guessed, thanks :)
Done!